packages/offix-cache/src/cache/mutations.ts

```diff
@@ -20,7 +20,7 @@ export interface MutationHelperOptions extends MutationOptions {
  * Provides useful helpers for cache updates, optimistic responses, and context
  * @param options see `MutationHelperOptions`
  */
-export const createMutationOptions = (options: MutationHelperOptions): MutationOptions => {
+export const createMutationOptions = (options: MutationOptions & MutationHelperOptions): MutationOptions => {
```

Comment:
ah.. that is even nicer. This will decorate MutationHelperOptions and still work with <T> depedency.